scripts/sonic-host-server

```diff
@@ -39,8 +26,7 @@ def register_dbus():
         'image_service': image_service.ImageService('image_service'),
         'docker_service': docker_service.DockerService('docker_service'),
         'file_stat': file_service.FileService('file'),
-        'debug_service': debug_service.DebugExecutor('DebugExecutor'),
-        'gnoi_reset': gnoi_reset.GnoiReset('gnoi_reset')
+        'debug_info': debug_info.DebugInfo('debug_info')
```

Comment:
Isn't debug_service different than debug_info ? Why are we removing debug_service ? 

gnoi_reset is missing here too.